modules/file/file.go

```diff
@@ -132,8 +142,11 @@ func (s *search) makeChecks() (err error) {
 			err = fmt.Errorf("makeChecks() -> %v", e)
 		}
 	}()
+	if s.Options.Debug == "print" {
+		debug = true
+	}
 	if s.Options.MaxDepth == 0 {
-		s.Options.MaxDepth = 10000
+		s.Options.MaxDepth = 1000
 	}
 	if s.Options.MatchLimit == 0 {
 		s.Options.MatchLimit = 1000
```

Comment:
That's 1,000 files returns in the results of a single search on a single host. It's possibly that people will run into this limit, but my guess is that it won't be intentional and only be the result of very large fingers :)